ts/Core/Legend.ts

```diff
@@ -1237,6 +1237,7 @@ class Legend {
              * @type {Highcharts.SVGElement}
              */
             legend.group = legendGroup = renderer.g('legend')
+                .addClass('highcharts-no-tooltip')
```

Comment:
Q: Should add an option `legend.className`, similar to annotation's label? So users can switch back to the old behaviour?

Reply:
Agreed. It could also be convenient for styling purposes.